### ISSUE TITLE
Update required for typescript-eslint-parser (v15) to support TypeScript 2.8 

### ISSUE THREAD

**Issue by pranavjindal999** [vuejs/vue-cli#1161]
### What problem does this feature solve?
[typescript-eslint-parser](https://github.com/eslint/typescript-eslint-parser) release [v15](https://github.com/eslint/typescript-eslint-parser/releases/tag/v15.0.0) now supports [TypeScript 2.8](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html).
- It will remove following `TS2.8` warning when serving/building

```
WARNING: You are currently running a version of TypeScript which is not officially supported by typescript-eslint-parser.
You may find that it works just fine, or you may not.
SUPPORTED TYPESCRIPT VERSIONS: ~2.7.1
YOUR TYPESCRIPT VERSION: 2.8.3
Please only submit bug reports when using the officially supported version.
```

### What does the proposed API look like?
No change. Just a dependency version upgrade.

**Comment by jkzing** [comment]
`cli-plugin-typescript` is using typescript@^2.7.2
I think `typescript-eslint-parser` v15 should be upgraded along with the typescript version.

**Comment by pranavjindal999** [comment]
It is v14 in my `@vue/eslint-config-typescript/package.json` . I think it won't auto update a major release, i.e. v15

![image](https://user-images.githubusercontent.com/5094327/39131646-9d6334c6-472d-11e8-9525-db45d6246136.png)

**Comment by jkzing** [comment]
I mean upgrade along with the typescript version in `cli-plugin-typescript`, not auto update...

**Comment by kunalkundaje** [comment]
It looks like this has been fixed in `v3.0.0-beta.7`
https://github.com/vuejs/vue-cli/commit/c9d7feffa136b206dc0780c378a0de58f788bd1d#diff-f2c6c93611681b3110b5ecc96977759eR25
